roles/postgres/tasks/main.yml

```diff
@@ -24,7 +24,7 @@
 - name: Obtain custom resource information
   set_fact:
     custom_resource: "{{ hostvars[inventory_hostname][custom_resource_key] }}"
-    custom_resource_status: "{{ hostvars[inventory_hostname][custom_resource_key]['status'] }}"
+    custom_resource_status: "{{ hostvars[inventory_hostname][custom_resource_key]['status'] | default({}) }}"
```

Comment:
You added the `default` filter to avoid breaking the execution in the first execution, correct?